<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pacman</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/pacman.png</normaloff>../img/pacman.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">stylesheet.qss</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="welcome_page">
     <property name="styleSheet">
      <string notr="true">stylesheet.qss</string>
     </property>
     <widget class="QPushButton" name="startGameButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>270</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
